### PR TITLE
[UI] MesheryFilters/FilterCard  : Add Download Button to Filter Card 

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

This PR fixes #CU-85ztd2m9f

Added Button to download wasm binary . 

image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-85ztd2m9f Create a download button for the wasm binaries](https://app.clickup.com/t/85ztd2m9f)

**Review by senali-d — COMMENTED** [review]
LGTM

**Comment by leecalcote** [comment]
@aabidsofi19 did you include support for dark mode, too?

File: ui/components/MesheryFilters/FiltersCard.js [hunk 1]
```diff
@@ -122,6 +124,15 @@ function FiltersCard({
                   </TooltipButton>
                 )
               }
+              <TooltipButton
+                title="Download"
+                variant="contained"
+                color="primary"
+                onClick={handleDownload}>
+
+                <DownloadIcon fill="#ffffff" className={classes.iconPatt} />
```
Comment (leecalcote) [inline ui/components/MesheryFilters/FiltersCard.js:133]: Fill color hard coded. Dark mode supported?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Does this grid's responsiveness include an update for this new button?

**Comment by leecalcote** [comment]
Related: Please make sure that the Clone button maintains aspect ratio on smaller screen widths, @aabidsofi19

**Comment by leecalcote** [comment]
@ritiksaxena124 is this the right set of colors?

**Comment by leecalcote** [comment]
@aabidsofi19 any reason that this button deviates from the existing icon? Let's avoid the potential confusion this will mean for users.
Screenshot 2023-07-06 at 1 35 50 PM

**Comment by aabidsofi19** [comment]
> @aabidsofi19 did you include support for dark mode, too?

working fine , btn turns teal and fill is white

**Comment by aabidsofi19** [comment]
> @aabidsofi19 any reason that this button deviates from the existing icon? Let's avoid the potential confusion this will mean for users.
> Screenshot 2023-07-06 at 1 35 50 PM
> 

This looked a bit better as this is a big btn , if it's confusing let's avoid it ..

**Comment by aabidsofi19** [comment]
@leecalcote  @@senali-d  I left the button grid unmodified since it has five columns. However, it's not fully responsive on mobile devices, causing the button text to overflow even without adding a new button. Considering that we will likely have a maximum of three buttons ("publish/unpublish," "download," and "clone"), I suggest using three columns for mobile devices.

**Comment by leecalcote** [comment]
@aabidsofi19 we will have more than three buttons. Make the card responsive. Don't allow spill over.

**Comment by aabidsofi19** [comment]
> @aabidsofi19 we will have more than three buttons. Make the card responsive. Don't allow spill over. 

@leecalcote  then should we truncate text on mobile as well  .

**Comment by leecalcote** [comment]
do it: remove the text

**Comment by aabidsofi19** [comment]
https://github.com/meshery/meshery/assets/65964225/1e8cab47-d3b5-435c-b241-bf6920cb232b

**Comment by ritiksaxena124** [comment]
@aabidsofi19 looks to me, small changes 
1. The color that we are using for the text "_Modified on: July 7, 2023 10:34 AM_" i.e., #607d8b is not our brand color could you update it to color Hoki (#677E88) please? Other colors used are good
2. On smaller screens the download icon is not in center

**Comment by aabidsofi19** [comment]
> @aabidsofi19 looks good to me, small changes
> 
> 1. The color that we are using for the text "_Modified on: July 7, 2023 10:34 AM_" i.e., #607d8b is not our brand color could you update it to color Hoki (#677E88) please? Other colors used are good
> 2. On smaller screens the download icon is not in center
> 3. Don't import icons from Material UI instead use the JS version of the SVG

3rd change will need refactoring of more than 10-15 components , lets do it in a separate pr .

**Comment by aabidsofi19** [comment]
@ritiksaxena124  we are using primary  for modifed on subheading :confused

**Comment by ritiksaxena124** [comment]
> @ritiksaxena124 we are using primary for modifed on subheading :confused

Yes, the Hexcode for primary needs to be updated

**Comment by aabidsofi19** [comment]
> > @ritiksaxena124 we are using primary for modifed on subheading :confused
> 
> Yes, the Hexcode for primary needs to be updated

in the theme/app.js

**Comment by aabidsofi19** [comment]
@ritiksaxena124 what should be the main , dark and light variant for it

**Comment by ritiksaxena124** [comment]
we are using blueGrey for both dark and light and we can replace it with "Hoki". it will work in both dark and light mode as the current color and hoki has a very slight difference

**Comment by aabidsofi19** [comment]
there is no Hoki color exported from mui : 
image

**Comment by aabidsofi19** [comment]
> 

@ritiksaxena124

**Comment by leecalcote** [comment]
> there is no Hoki color exported from mui :
> image

Don't use MUI colors, @aabidsofi19. Use our theme colors. Import the theme.

**Comment by aabidsofi19** [comment]
> > there is no Hoki color exported from mui :
> 
> Don't use MUI colors, @aabidsofi19. Use our theme colors. Import the theme.

@leecalcote  thats right ,@ritiksaxena124   asked to change the primary color in theme from blueGreen to "hoki" . But I can't find the color anywhere .. rest assured we are not importing colors from mui in these components .

**Comment by aabidsofi19** [comment]
> > @ritiksaxena124 we are using primary for modifed on subheading :confused
> 
> Yes, the Hexcode for primary needs to be updated

fixed it .

File: ui/themes/app.js
Comment (leecalcote) [inline ui/themes/app.js:1]: Please revert this, @aabidsofi19. Just use the existing colors.

**Pull request merged**